### PR TITLE
Add /pt-br/docs/reference/glossary/deployment

### PR DESCRIPTION
**Pull request by MrErlison**
Added translation for portuguese of the glossary term deployment.

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/deployment.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+short_description: >
+  Gerencia uma aplicação replicada no seu cluster.
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+---
+ Um objeto API que gerencia uma aplicação replicada, geralmente executando Pods sem estado armazenado.
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/deployment.md:15]: nit: objeto da API.

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @MrErlison!

Thank you very much for your contribution! 🎉 

I just have one minor nit, other than that it's a LGTM!

**Comment by edsoncelio** [comment]
Thanks!
/approved

**Pull request merged**